src/rpc.ts

```diff
@@ -284,7 +288,7 @@ export class RPC extends EventEmitter {
           type: 'reply',
           serviceID: RPC.serviceID,
           id: packet.id,
-          error: { code: 4003, message: 'Unknown method name' },
+          error: { code: 4003, message: `Unknown method name: ${packet.method}` },
```

Comment:
If you get a moment it'd be cool to update tetrisd as well. This string matches exactly what tetrisd returns normally.

Reply:
Looks like this is the error in tetrisd:
```
	case EAPP_BAD_METHOD:
                str = "Unknown method name \"%s\"."
```

I'll remove the colon but it already specifies the method that wasn't defined if I understand correctly?